sharespots/settings/test.py

```diff
@@ -13,7 +13,10 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql',
+        'HOST': os.environ.get('POSTGRES_HOST'),
+        'NAME': os.environ.get('POSTGRES_DB'),
```

Comment:
- How does this affect our local setup now? 
- Does this get overridden in prod & staging settings?
- If this change is needed please document how to update local setup and where to get these environment variables.

Reply:
1. There is a `pytest.ini` config file which makes sure that django is using `test.py` settings file 
2. I believe in prod & staging we are using `production.py` & `staging.py` file
3. I am going to update "The run tests.." section of the README